src/openrct2/world/Footpath.cpp

```diff
@@ -1315,21 +1315,21 @@ static int32_t footpath_is_connected_to_map_edge_recurse(
             continue;
         }
 
-        if (!(flags & (1 << 0)))
+        if (!(flags & FOOTPATH_CONNECTED_MAP_EDGE_IGNORE_QUEUES))
         {
             if (tileElement->AsPath()->IsQueue())
             {
                 continue;
             }
         }
 
-        if (flags & (1 << 5))
+        if (flags & FOOTPATH_CONNECTED_MAP_EDGE_UNKNOWN)
```

Comment:
Perhaps checking the callers of this function will reveal what this flag is for. You can then also replace this flag in the callers of this function.